md/performance-tuning.md

```diff
@@ -170,6 +170,23 @@ Setting a high `queueCapacity` limit means that more work can be queued, but can
 It is essential to ensure that the queue never becomes full (`queueCapacity` is never reached).  
 If the queue becomes full, the application restarts in order to force the engine to generate all work from the database. This means that work is lost.
 
+##### SQLServer
+
+When Bonita platform is under high volumetry on work execution and database transaction, sometimes when one work commits its data and next transaction try to access it, this information was not yet visible.
+
+This issue happens only when using Bonita and BDM XA resources ( **XAMultipleResource** ) and because the **transaction isolation level** is configured as **ALLOW_SNAPSHOT_ISOLATION** and **READ_COMMITTED_SNAPSHOT**. These isolation levels are mandatory to avoid a deadlock.
+
+To avoid the issue describe above, we add a **work execution delay** (100 ms by default) only in case of the database is **SQL Server** and if the **previous transaction has used a XAMultipleResource ( Bonita + BDM )**.
```

Comment:
```suggestion
To avoid the issue described above, by default, a **100 ms work execution delay** is added when the database is **SQL Server** and if the **previous transaction has used a XAMultipleResource ( Bonita + BDM )**.
```